src/post.cc

```diff
@@ -126,6 +126,8 @@ string post_t::payee_from_tag() const
 {
   if (optional<value_t> post_payee = get_tag(_("Payee")))
     return post_payee->as_string();
+  else if (optional<value_t> post_payee = get_tag("Payee"))
+    return post_payee->as_string();
```

Comment:
Assume someone has a Ledger journal and has made use of the special metadata tag `Payee`. With the changes in this PR they are able to make Ledger <HeavyGermanAccent>speak German</HeavyGermanAccent>.

Without [this change](https://github.com/ledger/ledger/pull/2243/commits/ac4f561fa2586fe0545abf4344474812b5aad951) their journal would no longer recognize the English `Payee` keyword they have in their journal as Ledger would look for the German equivalent, i.e. `Empfänger`.

Therefore I believe Ledger should always support and recognize the English name of special metadata tags.